text/0014-continue-traces.md

```diff
@@ -0,0 +1,62 @@
+* Start Date: 2022-09-26
+* RFC Type: feature
+* RFC PR: https://github.com/getsentry/rfcs/pull/14
+
+# Summary
+
+This RFC proposes a new way to continue a trace when creating nested transactions.
+
+# Motivation
+
+The current way we propagate `sentry-trace` and `baggage`, is to pass a correctly populated `TransactionContext` as the first argument to `startTransaction()`.
+
+```php
+use Sentry\Tracing\TransactionContext;
+use function Sentry\startTransaction;
+
+$transactionContext = TransactionContext::continueFromHeaders($sentryTraceHeader, $baggageHeader);
```

Comment:
On Java, it's a bit different but similar https://docs.sentry.io/platforms/android/performance/instrumentation/custom-instrumentation/#distributed-tracing
We've not implemented the `continueFromHeaders` method, [reason is here](https://github.com/getsentry/sentry-java/issues/1346).